lib/hutch/cli.rb

```diff
@@ -189,6 +189,12 @@ def parse_options(args = ARGV)
           Hutch::Config.pidfile = pidfile
         end
 
+        opts.on('--consumers', 'Show consumers information') do
+          load_app
```

Comment:
Initially I want to make `Dir.glob` through require paths and don't load rails app. But it adds separate load consumers logic.